react-app/src/components/AppSideBar/Footer.tsx

```diff
@@ -20,8 +21,13 @@ const Footer: React.FC = () => {
           "sm:mt-5"
         )}
       >
-        <p className={cn("sm:grow", "sm:text-left")}>© 2022 Likecoin</p>
-        <p className={cn("mt-6", "sm:mt-0")}>Get Token</p>
+        <p className={cn("sm:grow", "sm:text-left")}>
+          © {new Date().getFullYear()}{" "}
+          <LocalizedText messageID="footer.likecoin" />
```

Comment:
you can probably have `"footer.likecoin": "© {year} Likecoin",` in the json file and then use 
```
<LocalizedText messageID="footer.likecoin" messageArgs={{ year: new Date().getFullYear()}}/>
```